holoviews/core/options.py

```diff
@@ -778,17 +778,18 @@ class Compositor(param.Parameterized):
       This pattern specification could then be associated with the RGB
       operation that returns a single RGB matrix for display.""")
 
-    group = param.String(doc="""
+    group = param.String(allow_None=True, doc="""
        The group identifier for the output of this particular compositor""")
 
     kwargs = param.Dict(doc="""
        Optional set of parameters to pass to the operation.""")
 
+    transfer_options = param.Boolean(default=False, doc="""
+       Whether to transfer the options from the input to the output.""")
 
```

Comment:
I think this is fine but maybe the part that transfers parameters (i.e plot options) to the operation should be a separate flag. Maybe ``transfer_parameters``?